docs/source/getting_started/movement_dataset.md

```diff
@@ -19,7 +19,11 @@ To learn more about `xarray` data structures in general, see the relevant
 
 ## Dataset structure
 
-![](../_static/dataset_structure.png)
+```{figure} ../_static/dataset_structure.png
+:alt: movement dataset structure
+
+A schematic showing the structure of `poses` and `bboxes` datasets in `movement`.
```

Comment:
Suggestion for caption:

> An xarray dataset is a collection of several data arrays that share some axes. The schematic shows the data arrays that make up the `poses` and `bboxes` datasets in `movement`.

It is a bit more specific description, and I think the concept in the first sentence is worth hammering in. But I leave it up to you, it may be too lengthy.

Reply:
I like it, I'll just say "dimensions" instead of "axes", to stay within xarray terminology.